open-api/rest-catalog-open-api.yaml

```diff
@@ -114,7 +124,8 @@ paths:
                 },
                 "defaults": {
                   "clients": "4"
-                }
+                },
+                "capabilities": ["views","pagination","remote_signing"]
```

Comment:
@danielcweeks and I briefly discussed this offline last week and came up with that list. I think we should probably discuss whether `capabilities` just indicate the existence of certain endpoints or whether they can be a general indicator of something that the server supports.

`pagination` is a good example where we probably wouldn't need to indicate that a server supports this via `capabilities`, since a client can be informed about this by sending the `next-page-token` in a list response.